packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/resizingModule.tests.js

```diff
@@ -131,6 +131,39 @@ module('Resizing module', moduleConfig, () => {
         assert.deepEqual(resizingInstance.allowedTargets, ['video'], '\'allowedTargets\' option has been applied');
     });
 
+    QUnit.module('moduleResizing runtime change', () => {
+        test('to { enabled: true } should render resize frame and attach events (T1241439)', function(assert) {
+            const resizingInstance = new Resizing(this.quillMock, this.options);
+            this.attachSpies(resizingInstance);
+            resizingInstance.option('mediaResizing', { enabled: true });
+
+            assert.ok(resizingInstance.enabled, '\'enabled\' option has been applied');
+            assert.strictEqual(this.$element.find(`.${RESIZE_FRAME_CLASS}`).length, 1, 'There is resize frame element');
+            assert.ok(this.attachEventsSpy.calledOnce, 'events has been attached');
+            assert.ok(this.detachEventsSpy.notCalled, 'events hasn\'t detached');
+        });
+
+        test('to { enabled: false } should remove resize frame and detach events (T1241439)', function(assert) {
+            this.options.enabled = true;
+            const resizingInstance = new Resizing(this.quillMock, this.options);
+            this.attachSpies(resizingInstance);
+
+            resizingInstance.option('mediaResizing', { enabled: false });
+            assert.notOk(resizingInstance.enabled, '\'enabled\' option should be disabled');
+            assert.strictEqual(this.$element.find(`.${RESIZE_FRAME_CLASS}`).length, 0, 'There should be no resize frame element');
+            assert.ok(this.detachEventsSpy.calledOnce, 'Events should be detached once.');
+        });
+
+        test('should not cause errors when repeatedly disabling enabled option (T1241439)', function(assert) {
+            this.options.enabled = true;
+            const resizingInstance = new Resizing(this.quillMock, this.options);
+            resizingInstance.option('mediaResizing', { enabled: false });
+            resizingInstance.option('enabled', false);
+
+            assert.ok(true, 'No errors should occur when disabling multiple times');
```

Comment:
[isolated test]
In such cases it's better to use `try catch` syntax for handling possible error to not break neighbor tests

Reply:
[bba349a](https://github.com/DevExpress/DevExtreme/pull/27707/commits/bba349a51ff1a26fc7bee4b232459520d1a7e683)